products/dotnet.md

```diff
@@ -8,62 +8,53 @@ alternate_urls:
 title: .NET
 versionCommand: dotnet --version
 releasePolicyLink: https://dotnet.microsoft.com/platform/support/policy/dotnet-core
-changelogTemplate: https://github.com/dotnet/core/blob/main/release-notes/__CYCLE_SHORT_HAND__/__LATEST__/__LATEST__.md
+changelogTemplate: https://github.com/dotnet/core/blob/main/release-notes/{{"__LATEST__"|slice:0,3}}/__LATEST__/__LATEST__.md
```

Comment:
I think that relying on exact length of the "major + dot + minor" substring isn't exactly future proof - nowadays there is already 7.0 in the works, so 10.0 probably isn't that far in the future (and that's where this `|slice:0,3` stops working, because it will extract just "`10.`").

I have a better idea - use [`split`](https://shopify.github.io/liquid/filters/split/) to get the array of version components separated by dot `.`, then [`slice`](https://shopify.github.io/liquid/filters/slice/) to get the first 2 version components (`slice` works for both strings and arrays) and [`join`](https://shopify.github.io/liquid/filters/join/) the array of 2 components with dot `.` again. Like this:

```suggestion
changelogTemplate: https://github.com/dotnet/core/blob/main/release-notes/{{"__LATEST__"|split:'.'|slice:0,2|join:'.'}}/__LATEST__/__LATEST__.md
```

Reply:
I will check much better solution after fixing/removing all other product's shorthand parts.
Just give me 10 minutes i will return back this one and check a better solution

Reply:
btw could you have time for checking/review my other pr/mr s  ?

Reply:
I've just noticed that this `slice:0,3` pattern is also in **magento.md**:

https://github.com/endoflife-date/endoflife.date/blob/086b9e309fd5e681015d252432a17215654328c4/products/magento.md?plain=1#L6-L8

So it would also benefit from the solution I suggested above.

Reply:
to be honest i think it will take at least 100 more years for them to reach 10.x version but i will fix it too thx

Reply:
could you give +1 for the ones you checked {just comment +1 in related PRs)
 so i will merge them